frontend/components/actions/items-action.ts

```
@@ -279,7 +279,10 @@ export async function removeItem(id: string) {
   }
 }
 
-export async function addItem(prevState: {}, formData: FormData) {
+export async function addItem(
+  prevState: Record<string, never>,
+  formData: FormData,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)